### PR TITLE
update privileges.rst button/link names to match phpMyAdmin v5.1.1

### PR DESCRIPTION
**Pull request by joelmellon**
### Description

Please describe your pull request.

Fixes #

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

File: doc/privileges.rst [hunk 1]
```diff
@@ -28,22 +28,22 @@ Editing an existing user
...
+From the :guilabel:`User accounts` page, check the checkbox for the user you wish to
 remove, select whether or not to also remove any databases of the same name (if
 they exist), and click Go.
 
 Assigning privileges to user for a specific database
 ----------------------------------------------------
 
 Users are assigned to databases by editing the user record (from the
-:guilabel:`User accounts` link on the home page).
+:guilabel:`User accounts` tab on the home page).
```
Comment (mauriciofauth) [inline doc/privileges.rst:46]: It's actually a link, not a tab. Although some themes make these links look like tabs.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
Hi, @joelmellon. Thank you for your contribution.

Before we could merge your pull request, could you please add the `Signed-off-by` line in your commit? Or your can add a comment with this line in this PR and we'll add it for you. 

Example:
https://github.com/phpmyadmin/phpmyadmin/blob/a2014e26744639f2f1d45ba6694a3cabd95f067c/DCO#L37

**Comment by joelmellon** [comment]
> add a comment with this line in this PR and we'll add it for you

Thanks, I hadn't cloned the repo, just a quick edit in GitHub. I don't need credit but if you must:

Signed-off-by: Joel Mellon <joelgmellon@gmail.com>

File: doc/privileges.rst (hunk 1 above)
Comment (mauriciofauth) [reply doc/privileges.rst:46]:
```suggestion
:guilabel:`User accounts` link on the home page).
```

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**